src/kibana/plugins/discover/components/field_chooser/field_chooser.html

```diff
@@ -21,8 +21,11 @@
       <button
         ng-class="{ 'btn-default': !filter.active, 'btn-success': filter.active, 'hidden-xs': !showFields, 'hidden-sm': !showFields }"
         class="btn btn-xs btn-default pull-right discover-field-filter-toggle"
-        ng-click="showFilter = !showFilter">
-        <i class="fa fa-gear"></i>
+        ng-click="showFilter = !showFilter"
+        aria-label="Field Settings"
+        aria-haspopup="true"
+        aria-expanded="{{ showFilter ? 'true' : 'false' }}">
+        <i aria-hidden="true" class="fa fa-gear"></i>
```

Comment:
No I mean to say, if you put `[aria-label]` on something, do the child elements also need the `[aria-hidden]`, or is the label enough? Especially when there is only one child and it's an icon.

From what I can see, I see both in the PR, which one should it be though?
My initial instinct is both if screen readers suffer from any of the same fragmentation as browsers